src/constant/pages/index.ts

```diff
@@ -8,6 +8,24 @@ import { LINKS, products, STATIC_FILE_PATH } from '../global';
 import { v4 } from 'uuid';
 import { routes } from '..';
 
+const COMMUNITY_SECTION: {
+  communityBannerImage: string;
+  communityTitleLine: string;
+  communityDescriptionLine: string;
+  goalOfCommunity: string[];
+} = {
+  communityBannerImage: '/images/Community-Section-Image.svg',
+  communityTitleLine: 'Community For Everyone',
+  communityDescriptionLine:
+    'You excel where you’re supported. Connect with like-minded peers who share the same goal as you.',
+  goalOfCommunity: [
+    'Attend Tech Workshops',
+    'Connect with Like-minded Peers',
+    'Share your journey with others',
+    'Find accountability parter',
+  ],
+};
+
```

Comment:
this is not needed because it's only one component which will not change in data